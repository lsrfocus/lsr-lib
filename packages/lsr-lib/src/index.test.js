import { ScrollToTopOnNavigate } from './index';

it('exports as expected', () => {
  expect(ScrollToTopOnNavigate).toMatchSnapshot();
});
