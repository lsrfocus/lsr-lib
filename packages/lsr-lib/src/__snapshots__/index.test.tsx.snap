// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports as expected 1`] = `[Function]`;
